Update groundMaterial shading

PButcher opened the pull request:
Suppresses the following warning when using the component with A-Frame v0.7.0:

`THREE.MeshLambertMaterial: .shading has been removed. Use the boolean .flatShading instead.`

`.shading` has been removed and is now `.flatShading`, see: https://threejs.org/docs/index.html#api/materials/Material

PButcher committed:
Update groundMaterial shading

Suppresses the following warning: `THREE.MeshLambertMaterial: .shading has been removed. Use the boolean .flatShading instead.`

`.shading` has been removed and is now `.flatShading`, see: https://threejs.org/docs/index.html#api/materials/Material

feiss commented:
thank you for your help! 
`flatShading` is available from threejs r87, so environment component would be broken for users using aframe < v0.7. Could you make it so it is retro compatible with aframe < v0.7 too? thanks!

PButcher committed:
Update groundMaterial shading (backwards compatible)

Use .shading for ground MeshLambertMaterial in A-Frame < 0.7.0 and .flatShading for >= 0.7.0

PButcher commented:
Good point, done.

PButcher committed:
Comment typo
